frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -384,9 +384,11 @@ function ReaderHighlight:highlightFromHoldPos()
     end
 end
 
+--[[
```

Comment:
Why has that been commented? (it happens to break the testsuite, so something relies on it).